### PR TITLE
Category back button arrow pointing to left (bug 1159935)

### PR DESCRIPTION
**Pull request by atiqueahmedziad**

### CODE REVIEW

File: src/media/css/elements--nav.styl [hunk 1]
```diff
@@ -201,11 +201,13 @@ mkt-nav-child {
     border-width: 0 2px 0 0;
     cursor pointer
     display: block;
+    hambacker($greyscale-white);
+    hambacker-back();
     height: $nav--header-height;
     left: -1 * $nav--width;
     overflow: hidden;
+    padding: 0 0 0 12px;
```
Comment (mstriemer) [inline src/media/css/elements--nav.styl]: I'd bump the `12px` up to `22px` so that it lines up with the icons below it.

**Comment by mstriemer** [comment]
r+wc

Bump up the padding and everything looks good. I'll run it past sherlocked to make sure we're good for RTL and different browsers too.

**Comment by atiqueahmedziad** [comment]
Done :)

**Comment by mstriemer** [comment]
Looks good, thanks for the patch!

**Pull request merged**